source/frontend/src/features/contacts/list/__snapshots__/ContactListView.test.tsx.snap

```diff
@@ -1237,17 +1237,13 @@ exports[`Contact List View > matches snapshot 1`] = `
                     role="cell"
                     style="box-sizing: border-box; flex: 60 0 auto; min-width: 30px; width: 60px; justify-content: left; text-align: left; flex-wrap: wrap; align-items: center; display: flex;"
                     title=""
-                  >
-                    first
```

Comment:
It looks like the test may be failing your new isPersonSummary test and returning an empty string. Can you please fix the test model so that it is not ignored (and potentially add an invalid model for testing that does return an empty result).